client/src/rpc_client.rs

```diff
@@ -306,6 +306,35 @@ impl RpcClient {
         self.get_account(pubkey).map(|account| account.data)
     }
 
+    pub fn get_minimum_balance_for_rent_exemption(&self, data_len: usize) -> io::Result<u64> {
+        let params = json!([format!("{}", data_len)]);
```

Comment:
```suggestion
        let params = json!([data_len]);
```
We don't want to format this as a string, or it will fail to parse to a usize on https://github.com/solana-labs/solana/pull/6109/files#diff-3857de88b9c51a001730fa012a6951b2R307 below